templates/index.html

```diff
@@ -36,6 +36,7 @@
   {% include "takeovers/_openstack_security_takeover.html" %}
   {% include "takeovers/_14-04-esm.html" %}
   {% include "takeovers/_devops-iot-webinar_takeover.html" %}
```

Comment:
I am pretty sure we don't want both iot takeovers up at the same time... can you make sure?